rpcs3/Emu/Cell/lv2/sys_ss.cpp

```diff
@@ -146,7 +143,7 @@ s32 sys_ss_get_open_psid(vm::ptr<CellSsOpenPSID> psid)
 
 error_code sys_ss_appliance_info_manager(u32 code, vm::ptr<u8> buffer)
 {
-	sys_ss.warning("sys_ss_appliance_info_manager(code=0x%x, buffer=*0x%x)", code, buffer);
+	sys_ss.trace("sys_ss_appliance_info_manager(code=0x%x, buffer=*0x%x)", code, buffer);
```

Comment:
Don't listen to that dated developer guide, trace is simply put not logged which is not what we want. here you can use notice level.

Reply:
LMAO. Changed.